src/themes/creator.js

```diff
@@ -412,100 +477,126 @@ export default function createTheme(
     },
     typography: {
       font100: {
-        fontFamily: primitives.primaryFontFamily,
-        fontSize: '11px',
-        fontWeight: 'normal',
-        lineHeight: '16px',
-      },
-      font200: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '12px',
         fontWeight: 'normal',
         lineHeight: '20px',
       },
-      font250: {
+      font150: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '12px',
-        fontWeight: 500,
+        fontWeight: '500',
         lineHeight: '20px',
       },
-      font300: {
+      font200: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '14px',
         fontWeight: 'normal',
         lineHeight: '20px',
       },
-      font350: {
+      font250: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '14px',
         fontWeight: 500,
         lineHeight: '20px',
       },
-      font400: {
+      font300: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '16px',
         fontWeight: 'normal',
         lineHeight: '24px',
       },
-      font450: {
+      font350: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '16px',
         fontWeight: 500,
         lineHeight: '24px',
       },
+      font400: {
+        fontFamily: primitives.primaryFontFamily,
+        fontSize: '18px',
+        fontWeight: 'normal',
+        lineHeight: '28px',
+      },
+      font450: {
+        fontFamily: primitives.primaryFontFamily,
+        fontSize: '18px',
+        fontWeight: 500,
+        lineHeight: '28px',
+      },
+      // deprecated, maintained for backward compatibility, may be removed in the future
       font460: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '18px',
         fontWeight: 'normal',
         lineHeight: '24px',
       },
+      // deprecated, maintained for backward compatibility, may be removed in the future
       font470: {
         fontFamily: primitives.primaryFontFamily,
         fontSize: '18px',
         fontWeight: 500,
         lineHeight: '24px',
       },
-      font500: {
+      font550: {
```

Comment:
Why the `X00` >= 500 were removed? We should keep those and set to the `normal` weight of the fonts in `X50`

Reply:
they aren't used anywhere and aren't in DS mockups, similar to new colors, so i figured to leave them out.

Reply:
Sounds good. It adds up to the migration but the codemod should help here 👍